portable/IAR/AtmelSAM7S64/lib_AT91SAM7X256.h

```diff
@@ -1,1469 +1,1396 @@
-//* \brief Set Reset Mode
-//*----------------------------------------------------------------------------
-__inline void AT91F_RSTSetMode(
-        AT91PS_RSTC pRSTC,
-        unsigned int mode)
-{
-    pRSTC->RSTC_RMR = (0xA5000000 | mode);
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_RSTGetMode
-//* \brief Get Reset Mode
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_RSTGetMode(
-        AT91PS_RSTC pRSTC)
-{
-    return (pRSTC->RSTC_RMR);
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_RSTGetStatus
-//* \brief Get Reset Status
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_RSTGetStatus(
-        AT91PS_RSTC pRSTC)
-{
-    return (pRSTC->RSTC_RSR);
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_RSTIsSoftRstActive
-//* \brief Return !=0 if software reset is still not completed
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_RSTIsSoftRstActive(
-        AT91PS_RSTC pRSTC)
-{
-    return ((pRSTC->RSTC_RSR) & AT91C_RSTC_SRCMP);
-}
+*               SOFTWARE API FOR PMC
+***************************************************************************** */
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PMC_CfgSysClkEnableReg */
+/** \brief Configure the System Clock Enable Register of the PMC controller */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PMC_CfgSysClkEnableReg( AT91PS_PMC pPMC, /* \arg pointer to PMC controller */
+                                                unsigned int mode )
+    {
+        /** Write to the SCER register */
+        pPMC->PMC_SCER = mode;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PMC_CfgSysClkDisableReg */
+/** \brief Configure the System Clock Disable Register of the PMC controller */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PMC_CfgSysClkDisableReg( AT91PS_PMC pPMC, /* \arg pointer to PMC controller */
+                                                 unsigned int mode )
+    {
+        /** Write to the SCDR register */
+        pPMC->PMC_SCDR = mode;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PMC_GetSysClkStatusReg */
+/** \brief Return the System Clock Status Register of the PMC controller */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PMC_GetSysClkStatusReg( AT91PS_PMC pPMC /* pointer to a CAN controller */
+                                                        )
```

Comment:
Should this be one line?

Reply:
Answered [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768#discussion_r1313523527)